spec/game_spec.rb

```diff
@@ -3,15 +3,16 @@
 
 RSpec.describe Game do
   describe "#run" do
-    it "displays an empty board" do
+    it "displays an empty numbered board" do
```

Comment:
> I'm also feeling like my acceptance test and unit test is really similar and I'm trying to figure out if I'm missing something or if it's just because all my Game does right now is return the Board's grid.

Could this be somewhat attributable to the specs having similar descriptions? Since this one is a feature spec, it may make sense for its description to be something like "plays tic tac toe," so the focus isn't on the detail of a numbered grid.

Reply:
I like that!